server/jwt.go

```diff
@@ -132,7 +132,9 @@ func validateTrustedOperators(o *Options) error {
 		if o.TrustedKeys == nil {
 			o.TrustedKeys = make([]string, 0, 4)
 		}
-		o.TrustedKeys = append(o.TrustedKeys, opc.Issuer)
+		if !opc.StrictSigningKeyUsage {
```

Comment:
So during a config reload, this function is invoked, but prior to that the o.TrustedKeys slice is set to nil, so this code would work. However, I am wondering what happens to s.trustedKeys, I am not sure this is handled at all. (same with the new map you are adding). I am not sure if this is supposed to be supported with a config reload, but just wondering if this is not putting things into a bad state.

Reply:
It's my understanding that operators won't be reloaded. 
processTrustedKeys which is called in NewServer copies trusted keys from options into the server.
(I made a change to make sure to make a deep copy at this place)

What is set to nil are the old options but processTrustedKeys and thus copying them over is not called.

Reply:
validateOptions() is called on reload. This in turn calls validateTrustedOperators(), so o.TrustedKeys will be reconstructed (because it is set to nil on reload), but s.trustedKeys is not. It may be ok if we don't support that, but wanted to make sure that this is not going to cause problems.

Reply:
I believe this is a side effect of how the code is structured and reloading keys or the operator is not supported.